src/data/sparse_page_source.h

```diff
@@ -312,9 +327,7 @@ class SparsePageSource : public DataSource<T> {
       size_t bytes_write = 0;
       double tstart = dmlc::GetTime();
       for (auto& batch : src->GetBatches<SparsePage>()) {
-        if (page_type == ".row.page") {
```

Comment:
Do we still have `row.page`?

Reply:
There is another method `CreateRowPage()` that creates CSR pages.